### PR TITLE
Replace all Unwraps with Expect

### PR DESCRIPTION
**Pull request by dmalis18**
### What does this PR do?

This PR is the first step removing all unwraps and expects from lading. The goal of this PR is to first turn each unwrap into an expect so a user can at least get feedback when the program panics. The next step would be to completely remove expects by handling errors properly

### Motivation

In general it is not a good idea for the program to panic and it is much better stylistically if the program handles errors instead.

### Related issues

SMPTNG-114

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: integration/ducks/src/main.rs [hunk 1]
```diff
@@ -323,7 +341,9 @@ async fn main() -> Result<(), anyhow::Error> {
     debug!("hello from ducks");
 
     // Every ducks-sheepdog pair is connected by a unique socket file
-    let ducks_comm_file = std::env::args().nth(1).unwrap();
+    let ducks_comm_file = std::env::args()
+        .nth(1)
+        .expect("Error: The length of env args is less than 2");
```
Comment (goxberry) [inline integration/ducks/src/main.rs:346]: Could be worth rephrasing as "Error: ducks socket file argument missing" to provide more context.

File: lading/src/bin/lading.rs [hunk 2]
```diff
@@ -80,13 +80,18 @@ impl FromStr for CliKeyValues {
         //
         // The approach taken here is to use the key notion as delimiter and
         // then tidy up afterward to find values.
-        static RE: Lazy<Regex> = Lazy::new(|| Regex::new(r"([[:alpha:]_]+)=").unwrap());
+        static RE: Lazy<Regex> = Lazy::new(|| {
+            Regex::new(r"([[:alpha:]_]+)=").expect("Error: Invalid regex pattern provided")
```
Comment (blt) [inline lading/src/bin/lading.rs:84]: I believe the "Error:" in these changes are redundant. The panic handler will signal something to the effect of a bad thing has happened.

**Review by blt — APPROVED** [review]
I'd prefer to remove the redundant `Error:` from the excepts here, but otherwise this looks good to me.

File: integration/ducks/src/main.rs (hunk 1 above)
Comment (dmalis18) [reply integration/ducks/src/main.rs:346]: I agree. I will make that fix when I make the other changes.

**Review by blt — APPROVED** [review]

File: integration/ducks/src/main.rs [hunk 3]
```diff
@@ -77,8 +77,16 @@ impl From<&HttpCounters> for HttpMetrics {
         HttpMetrics {
             request_count: val.request_count,
             total_bytes: val.total_bytes,
-            median_entropy: val.entropy.quantile(0.5).unwrap().unwrap_or_default(),
-            median_size: val.body_size.quantile(0.5).unwrap().unwrap_or_default(),
+            median_entropy: val
+                .entropy
+                .quantile(0.5)
+                .expect("Error: quantile is outside of the valid range")
```
Comment (goxberry) [inline integration/ducks/src/main.rs]: I'd suggest "Error: quantile argument must be between 0.0 and 1.0 inclusive", to provide more context.

**Review by goxberry — APPROVED** [review]

**Comment by GeorgeHahn** [comment]
> 🚂 MergeQueue
> 
> This merge request is not mergeable yet, because of pending checks/missing approvals. It will be added to the queue as soon as checks pass and/or get approvals. **Note:** if you pushed new commits since the last approval, you may need additional approval. You can remove it from the waiting list with `/remove` command.
> 
> Use `/merge -c` to cancel this operation!

Huh, that's new to me. @dmalis18 we use the plain Github UI rather than devflow. (Either `Squash and merge` or `Enable auto-merge (squash)`)

It looks like devflow added a `devflow/merge` step that presumably isn't ever going to show ready. To merge this, you'll need to check `Merge without waiting for requirements to be met (bypass branch protections)` and then `Squash and merge`. We wouldn't do that normally because it means CI is still running or has failed.

**Pull request merged**